src/guide/introduction.md

```diff
@@ -10,6 +10,20 @@ footer: false
 - Upgrading from Vue 2? Check out the [Migration Guide](https://v3-migration.vuejs.org/).
   :::
 
+<div class="vue-mastery-link">
```

Comment:
Instead of importing `vue-mastery.css` in `theme/styles/index.css`, it's better to import it here (via `<style src="...">`) so it's collocated with the markup, which makes it easier to remove if needed.